specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -1095,6 +1128,156 @@
         }
       }
     },
+    "statusBanner": {
+      "description": "banner type of emerging issue",
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "the banner title."
+        },
+        "message": {
+          "type": "string",
+          "description": "the details of banner."
+        },
+        "cloud": {
+          "type": "string",
+          "description": "the cloud type of this banner."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "description": "the last time modified on this banner.",
+          "format": "date-time"
+        }
+      }
+    },
+    "impactedRegion": {
+      "description": "object of impacted region",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "the impacted region slug id."
+        },
+        "name": {
+          "type": "string",
+          "description": "the impacted region name."
+        }
+      }
+    },
+    "emergingIssueImpact": {
+      "type": "object",
+      "description": "object of impacted region",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "the impacted region slug id."
+        },
+        "name": {
+          "type": "string",
+          "description": "the impacted region name."
+        },
```

Comment:
From the sample this seems not the id/name of the region.

Reply:
This is the id/name of the impacted services. I'll change the description here. Thanks for the catch. : )

Reply:
fixed